tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
The filename for this test class should be `configWpHome.php`. See [Writing PHPUnit Tests - Test Classes](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes).

Reply:
renamed